.env.example

```diff
@@ -6,42 +6,15 @@ APP_NAME=Meetable
 # Provide the base URL (no trailing slash) to where you've deployed the app
 APP_URL=https://events.example.org
 
-# Provide a URL to a logo to use in the header bar
```

Comment:
They are added as part of the setup, which can now define options which are missing.

Reply:
Right, but you can't take these out of the example config file because that's what someone is going to read when they are deploying this *not* on Heroku.

Reply:
It sounds like there are two conflicting goals.

The installer runs on all environments; yes or no?
It's a desire to reduce the places a user sets up the system; yes or no?

I can understand removing some of app.json, but it seems like .env or app.json or installer is three systems to maintain and a step too far. The reasons for app.json being unsuitable apply moreso to dotenv.

Another wrinkle is that as a Laravel developer on shared hosting, I might edit the config files.

What is it that makes dotenv so important? It seems like yet another silo (as I accept heroku is).

Should the goal not be the installer not having a filesystem dependency on this?

Reply:
I think my mistake was moving application settings into this config file in the first place rather than using a web GUI for it and storing them in the database. The reason we need this installer at all is because Laravel won't run at all unless certain things are defined in the `$_ENV`, so we need some way for the user to set that up. Really we should have only the minimum options needed for the software to boot up defined this way, and everything else like the logo and stuff can be configured in the app settings after they've logged in. I'm going to go ahead and try to knock out #63 right now and remove a bunch of stuff from the `.env.template` file to hopefully simplify the rest of this discussion.

Reply:
Thank you